### DISCUSSION TITLE
custom template can add systems param ? for example I need System.currentTimeMillis to get token

### DISCUSSION THREAD

**Discussion by xiaoguolong** [apache/hertzbeat#1319] [General]
有个场景，自定义模板需要传时间戳去获取token，进行下一步操作；时间戳有直接获取的吗，类似用占位符 $.currentTimeMillis获取当前系统的时间

**Comment by tomsun28** [comment]
试试函数 `now()` https://www.yuque.com/boyan-avfmj/aviatorscript/ashevw

**Reply by xiaoguolong** [reply]
尝试加了now() 模板不能识别 ，直接不显示了
![image](https://github.com/dromara/hertzbeat/assets/33684988/a67deeeb-f6be-4b8a-a37c-9c83c10449fa)
![image](https://github.com/dromara/hertzbeat/assets/33684988/8721a04c-31f0-4a4a-b489-095d8c571af6)
